icu4j/main/tests/core/src/com/ibm/icu/dev/test/format/DateIntervalFormatTest.java

```diff
@@ -545,7 +545,7 @@ public void TestFormat() {
 
                 "de", "2007 10 10 10:10:10", "2007 11 10 10:10:10", "EEEEdMMM", "Mittwoch, 10. Okt. \\u2013 Samstag, 10. Nov.",
 
-                "de", "2007 10 10 10:10:10", "2007 11 10 10:10:10", "EdMy", "Mi., 10.10.2007 \\u2013 Sa., 10.11.2007",
+                "de", "2007 10 10 10:10:10", "2007 11 10 10:10:10", "EdMy", "Mi., 10.10. \\u2013 Sa., 10.11.2007",
```

Comment:
I take on faith that these changes (and in C++) are intentional

Reply:
They were intentional on the part of the vetters. They make sense, other locales compress the ranges in this way.